### ISSUE TITLE
Spiders with small hitboxes bug out on certain blocks

### ISSUE THREAD

**Issue by solarsonata** [TheCyberBrick/Spiders-2.0#34]
https://imgur.com/Jo2bGTf
Not sure whether this is just for modded spiders.

**Comment by TheCyberBrick** [comment]
Yeah, that's currently an issue with small bounding boxes. Haven't found a reasonable fix yet

**Comment by solarsonata** [comment]
ah, well good luck in looking for one... at worst you can just say it's a feature :p
